unstructured_inference/inference/layoutelement.py

```diff
@@ -321,13 +327,72 @@ def get_elements_from_ocr_regions(ocr_regions: List[TextRegion]) -> List[LayoutE
             r.y1,
             r.x2,
             r.y2,
+            None,
             text=r.text,
             type="UncategorizedText",
         )
         for r in merged_regions
     ]
 
 
+def probably_contained(
+    element: Union[LayoutElement, Rectangle],
+    inside: Union[LayoutElement, Rectangle],
+    area_threshold: float = 0.9,
+) -> bool:
+    """This function checks if one element is inside other.
+        If is definetly inside returns True, in other case check
+    if the intersection is big enough."""
+    # Probably can be implemented usigng is_in, however this is based in area overlapping
+    if element.is_in(inside):
+        return True
+
+    intersected_area = element.intersection(inside)
+    if intersected_area:
+        return intersected_area.area >= inside.area * area_threshold
+    return False
```

Comment:
This is the core, well one of, of this PR it seems.
area overlap seem ok but may have recall issues: e.g., the boxes "Word", "oil" are ok to be not merged (if they were to be merged we might have other issues. But the box "Emerging" seem like a case it should be because the box bounding the text has high overlap on width and height: the point is area is width * height so suppose both sides got 0.9 overlap but the area would be only 0.81 overlap at best. 
![Screenshot 2023-09-15 at 10 47 58 AM](https://github.com/Unstructured-IO/unstructured-inference/assets/647930/080a3ac1-06ea-48f6-a0c8-de729726bc71)